DigitalExcellence/dex-frontend
# feature/361-implement-wizard

## Description

This pull request implements the designs and functionality of the manual wizard. It also includes the public flow from github and gitlab. 

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Click the add project button in the header
2. Try to click an external source without logging in
3. There should be an error
4. Now login and click GitHub
5. Import this repository
6. The expected data should be imported
7. Go through the steps and add some data in every step
8. Click finish
9. The project should be added to the database
10. Repeat steps 6 - 9 with all sources
11. Repeat steps 7 - 9 with the manual flow

## Link to issue

Closes: #361 
Fixes: #431
